pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -813,6 +817,32 @@ cdef class _Timedelta(timedelta):
         else:
             return "D"
 
+    @property
+    def resolution_timedelta(self):
+        """
+        Return a timedelta object (rather than a string)
+        representing the lowest resolution we have.
+        to retrieve a string use the resolution property.
```

Comment:
Instead, could you create a "See Also" section with the new method name, similar to this: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.isin.html